packages/manager/modules/pci/src/projects/project/training/jobs/info/logs/logs.controller.js

```diff
@@ -0,0 +1,25 @@
+export default class PciTrainingJobsInfoLogsController {
+  /* @ngInject */
+  constructor($interval, PciProjectTrainingJobService) {
+    this.$interval = $interval;
+    this.pciProjectTrainingJobService = PciProjectTrainingJobService;
+  }
+
+  $onDestroy() {
+    if (this.interval !== null) {
+      this.$interval.cancel(this.interval);
+    }
+  }
+
+  $onInit() {
+    if (this.job.isRunning()) {
+      this.interval = this.$interval(() => {
+        this.pciProjectTrainingJobService
+          .logs(this.projectId, this.jobId)
+          .then((jobLog) => {
+            this.jobLog = jobLog;
+          });
+      }, 3000);
```

Comment:
Why 3000 ?

Reply:
Seems like the 3s is the polling default on the project :), search for `, 3000` you will see a lot :D